br/tests/br_tiflash_conflict/prepare_data/insert_data.sql

```diff
@@ -0,0 +1,35 @@
+-- Switch to the test database
```

Comment:
Also you may add license header in each file.

Reply:
Seems that most .sql don't have a license header?